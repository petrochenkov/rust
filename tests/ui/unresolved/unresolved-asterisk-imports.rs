use not_existing_crate::*; //~ ERROR unresolved import `not_existing_crate
use std as foo;
//~^ ERROR cannot determine resolution for the import
//~| ERROR cannot determine resolution for the import

fn main() {}
