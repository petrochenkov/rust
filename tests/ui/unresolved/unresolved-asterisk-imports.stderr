error: cannot determine resolution for the import
  --> $DIR/unresolved-asterisk-imports.rs:2:5
   |
LL | use std as foo;
   |     ^^^^^^^^^^

error: cannot determine resolution for the import
  --> $DIR/unresolved-asterisk-imports.rs:2:5
   |
LL | use std as foo;
   |     ^^^^^^^^^^
   |
   = note: duplicate diagnostic emitted due to `-Z deduplicate-diagnostics=no`

error[E0432]: unresolved import `not_existing_crate`
  --> $DIR/unresolved-asterisk-imports.rs:1:5
   |
LL | use not_existing_crate::*;
   |     ^^^^^^^^^^^^^^^^^^ use of unresolved module or unlinked crate `not_existing_crate`
   |
help: you might be missing a crate named `not_existing_crate`, add it to your project and import it in your code
   |
LL + extern crate not_existing_crate;
   |

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0432`.
